### PR TITLE
Update 4.14 ec tests to 4.15 ec

### PR DESCRIPTION
**Pull request by svetsa-rh**
Moving from 4.14 ec to 4.15 ec. Confirmed with Roshni.
https://issues.redhat.com/browse/PERFSCALE-2550

### CODE REVIEW

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 1]
```diff
@@ -3109,22 +3120,11 @@ periodics:
...
-      channel: '#ocp-qe-scale-ci-results'
-      job_states_to_report:
-      - success
-      - failure
-      - error
-      report_template: '{{if eq .Status.State "success"}} :white_check_mark: Job *{{.Spec.Job}}*
-        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :white_check_mark:
-        {{else}} :warning:  Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
-        logs> :warning: {{end}}'
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-nightly-x86-perfscale-rosa-multiaz-node-density-cni-24nodes-periodic
```
Comment (krishvoor) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:3127]: Please slack's reporter config:
```
  reporter_config:
    slack:
      channel: '#ocp-qe-scale-ci-results'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :white_check_mark: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :white_check_mark:
        {{else}} :warning:  Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> :warning: {{end}}'
     
```

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 2]
```diff
@@ -3295,22 +3295,11 @@ periodics:
...
-      channel: '#ocp-qe-scale-ci-results'
-      job_states_to_report:
-      - success
-      - failure
-      - error
-      report_template: '{{if eq .Status.State "success"}} :white_check_mark: Job *{{.Spec.Job}}*
-        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :white_check_mark:
-        {{else}} :warning:  Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
-        logs> :warning: {{end}}'
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-ec-x86-perfscale-rosa-multiaz-control-plane-24nodes-periodic
```
Comment (krishvoor) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:3302]: Here as well, please add slack reporter_config

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 3]
```diff
@@ -3470,22 +3459,11 @@ periodics:
...
-      channel: '#ocp-qe-scale-ci-results'
-      job_states_to_report:
-      - success
-      - failure
-      - error
-      report_template: '{{if eq .Status.State "success"}} :white_check_mark: Job *{{.Spec.Job}}*
-        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :white_check_mark:
-        {{else}} :warning:  Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
-        logs> :warning: {{end}}'
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-ec-x86-perfscale-rosa-singleaz-data-path-2nodes-periodic
```
Comment (krishvoor) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:3466]: Here as well, please add slack reporter_config

**Review by krishvoor — CHANGES_REQUESTED** [review]
Needs slack_reporter_config changes

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-ec-x86-perfscale-rosa-multiaz-node-density-heavy-24nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-ec-x86-perfscale-rosa-singleaz-data-path-2nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.15-ec-x86-perfscale-rosa-multiaz-control-plane-24nodes-periodic

**Comment by svetsa-rh** [comment]
@krishvoor FYI, it turns out 4.15 ec is not available for Classic Rosa yet. Also, I am having rebase issues with this PR. Hence, I will close this PR and once we have support for 4.15 ec, will open a new PR.
For now, will continue to run the tests with 4.14 ec.

**Pull request closed without merging**